src/integration-test/tests/legal/amountClaim/pages/details-summary.ts

```diff
@@ -1,26 +1,20 @@
-'use strict'
-/* global actor */
-const verifyPageData = require('../../../../data/legal-test-data').verifyPageData
+import I = CodeceptJS.I
 
-let I
+const I: I = actor()
 
-module.exports = {
+import { verifyPageData } from 'integration-test/data/legal-test-data'
```

Comment:
I would love to move it to the top so we have imports first and constants later.